analysis_options.yaml

```diff
@@ -45,4 +45,14 @@ linter:
 
     join_return_with_assignment: false
 
-    prefer_const_literals_to_create_immutables: false
\ No newline at end of file
+    prefer_const_literals_to_create_immutables: false
+
+
+    ####################################################
+    #### Added to make things work for now. Must be ####
+    ############# removed in future PRs ################
```

Comment:
Please be more specific in what lines need to be restored so that someone working to restore functionality knows exactly want to do.

Possibly put the lines that need to be restored in this comment block

Reply:
@palisadoes done. Please look at the added comments and see whether they will help a new comer to solve the issues.